frontend/src/pages/ContributionView.jsx

```
@@ -0,0 +1,286 @@
+  const course = contentData[exampleId];
+
+  return (
+    <div className="contribution-view">
+      <Navbar />
+      
+      {/* Breadcrumb Navigation */}
+      <div className="breadcrumb">
+        <button 
+          onClick={() => setIsMenuHidden(!isMenuHidden)}
+          className="menu-toggle-btn"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)